articles/architecture-scenarios/_includes/_authentication/_m2m.md

```diff
@@ -0,0 +1,15 @@
+There are many scenarios that require an application that is not a user-interactive session to get an access token.  In these scenarios you are authenticating the client instead of the user.  Some common times where this is needed:
```

Comment:
```suggestion
There are many scenarios that require an application _without_ any user-interactive session to obtain an access token in order to call an API, and in such cases you must authenticate the client instead of the user. Some common examples of where this is needed include:
```